### DISCUSSION TITLE
Tweak rejuv settings

### DISCUSSION THREAD

**Discussion by elninost0rm** [isid0re/SoloLeveling#109] [Ideas]
Hi there,

My two hammerdins were roughly ~75/76 in Hell.  I noticed they were barely progressing and chickening constantly, but it's mainly because they have no rejuvs mapped in their belt.

I switched column for in the "hp hp mp mp" setting to rv and it's been smooth sailing since then.

Just a suggestion.  I find that 8 mana potions is too much at 70+, even if the toon doesn't have a merc that has Insight yet.

**Comment by isid0re** [comment]
Thanks for this.
